rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2504,6 +2504,17 @@ public long getNextPegoutCreationBlockNumber() {
         return activations.isActive(RSKIP271) ? provider.getNextPegoutHeight().orElse(0L) : 0L;
     }
 
+    public int getQueuedPegoutsCount() {
+        if (activations.isActive(RSKIP271)) {
+            try {
+                return provider.getReleaseRequestQueue().getEntries().size();
```

Comment:
Maybe we could a method in BridgeStorageProvider to return the release request queue size. What do you think? A bit nitpicky maybe, but I believe we are violating Demeter's law here

Reply:
Done